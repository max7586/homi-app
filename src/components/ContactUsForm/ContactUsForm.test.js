import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import ContactUsForm from './ContactUsForm';
import { BrowserRouter } from 'react-router-dom'

describe('ContactUsForm component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(
        <BrowserRouter>
        <ContactUsForm />
      </BrowserRouter>, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the UI as expected', () => {
    const tree = renderer.create(<ContactUsForm />).toJSON();
    expect(tree).toMatchSnapshot();  
  });


});