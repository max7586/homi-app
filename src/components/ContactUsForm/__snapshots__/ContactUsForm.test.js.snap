// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactUsForm component renders the UI as expected 1`] = `
<form
  className="contactme__form"
>
  <p>
    Fill in the next form to send us a message
  </p>
  <div>
    <label
      htmlFor="name"
    >
      Name
    </label>
    <input
      className="contactme__form__input"
      name="name"
      onChange={[Function]}
      placeholder="Your name here please"
      type="text"
    />
  </div>
  <div>
    <label
      htmlFor="email"
    >
      Email
    </label>
    <input
      name="email"
      onChange={[Function]}
      placeholder="We will contact you after reviewing your message"
      type="email"
    />
  </div>
  <div>
    <label
      htmlFor="name"
    >
      Message
    </label>
    <textarea
      maxLength="450"
      onChange={[Function]}
    />
  </div>
  <div>
    <p>
      We will answer as soon as possible
    </p>
    <input
      name="submit"
      onClick={[Function]}
      type="submit"
      value="Send"
    />
  </div>
</form>
`;
